resources/db/schema/mysql-ddl.sql

```diff
@@ -460,14 +460,13 @@ DROP TABLE IF EXISTS missed_parts;
 CREATE TABLE missed_parts (
   id       INT(16) UNSIGNED NOT NULL AUTO_INCREMENT,
   numberid BIGINT UNSIGNED  NOT NULL,
-  group_id INT(11) UNSIGNED NOT NULL DEFAULT '0'
-  COMMENT 'FK to groups',
+  group_id INT(11) UNSIGNED NOT NULL DEFAULT '0' COMMENT 'FK to groups.id',
```

Comment:
Only the line formatting is changed, it's still group_id not groups_id. I intend to do all tpg tables as a separate patch, as it needs to handle both tpg and non-tpg cases. We should just make TPG the default ;-)